slither/__main__.py

```diff
@@ -228,9 +225,7 @@ def main_impl(all_detector_classes, all_printer_classes):
 
 
 def parse_args(detector_classes, printer_classes):
-    parser = argparse.ArgumentParser(description='Slither',
-                                     usage="slither.py contract.sol [flag]",
-                                     formatter_class=lambda prog: argparse.HelpFormatter(prog, max_help_position=35))
+    parser = argparse.ArgumentParser()
```

Comment:
Nitpick - I realize this is a WIP but the current output doesn't include the `Slither` description/title and we are duplicating arguments. 

<img width="609" alt="screen shot 2018-10-22 at 5 05 20 pm" src="https://user-images.githubusercontent.com/13700823/47320441-6add0500-d61f-11e8-8fae-218ab3c3d474.png">

Seems like an easy fix. We could utilize the same parameters from before and pass them into our parser. Our output would be more consistent with what we had before and it removes that duplication:
```python
    parser = argparse.ArgumentParser(description='Slither',
                                     usage="slither.py contract.sol [flag]")
```
<img width="566" alt="screen shot 2018-10-22 at 5 29 45 pm" src="https://user-images.githubusercontent.com/13700823/47320672-1b4b0900-d620-11e8-91ae-0aa2ec628041.png">

This approach tends to stay consistent with other CLIs. Our messaging isn't ideal in the case that someone passes in a bad arguement (eg `abc12345`) but we still surface a useful message. Don't think we need to kill ourselves anyway for optimizing to that use case.

Reply:
thx for the feedback, fixed